package dough_variance_msgs.msgs;


message DoughVariance
{
    
    required bool variance = 1;
    required float dough_base_height = 2;

}
